addons/mail/static/src/new/user_settings_service.js

```diff
@@ -57,6 +60,12 @@ class UserSettings {
         return constraints;
     }
 
+    async performRpcSetResUsersSettings(resUsersSettings) {
+        await this.orm.call("res.users.settings", "set_res_users_settings", [[this.id]], {
+            new_settings: resUsersSettings,
+        });
+    }
+
```

Comment:
Method makes a useless indirection. Do not introduce this method and do `orm.call` immediately where needed